### PR TITLE
Space Travelers' Hub

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
This application will allow users to book rockets and join selected space missions. It is a web application for a company that provides commercial and scientific space travel services. It uses real live data from the SpaceX API. In this project we implemented the following functionalities.

- Created a rocket page that displays available SpaceX rockets.
- Rocket reservation and cancelation functionality.
- Created a dragon page that displays available SpaceX rockets.
- Dragon reservation and cancelation functionality.
- Created a mission section that displays a list of current missions.
- Implemented join and leave mission functionality.
- Created profile interface that displays a list of reserved rocked and dragons and joined missions.

### CODE REVIEW

File: src/components/navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <div>
+      <header className="panel-bg">
+        <div className="navigation">
+          <div className="logoFrame">
+            <img className="logo" src="/planet.png" alt="space logo" />
+            <span className="Logo-Text">Space Travelers Hub</span>
+          </div>
+          <ul>
+            <li>
+              <Link to="/" className="Mission Text-Style-3">
```
Comment (topeogunleye) [inline src/components/navbar/Navbar.js:41]: - [ ] As required by project requirements, I kindly suggest that you use `Navlink`. The Link component is used to navigate the different routes on the site. But `NavLink` is used to add the style attributes to the active routes.
`NavLink` will make it easier for you to style your Links and it is a requirement for active links to have an underline.
- [ ] As required. I kindly suggest that your current active page is indicated with an underline in the `NavLink`.



| Your Design | Figma Design |
| ------------- | ------------- |
| image  | image  |

```
 <ul>
        <li>
          <NavLink exact activeClassName="active" to="/">
            Home
          </NavLink>
        </li>
        <li>
          <NavLink activeClassName="active" to="/users">
            Users
          </NavLink>
        </li>
        <li>
          <NavLink activeClassName="active" to="/contact">
            Contact
          </NavLink>
        </li>
      </ul>
```

File: src/components/dragon/dragon.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                  : dragon.flickr_images
+              }
+              alt="dragon"
+              className="img-fluid"
+            />
+          </Col>
+          <Col md={8}>
+            <h3>{dragon.name}</h3>
+            <p>
+              <Badge bg="primary">{dragon.reserved ? 'Reserved' : ''}</Badge>{' '}
```
Comment (topeogunleye) [inline src/components/dragon/dragon.js:39]: - [ ] [OPTIONAL] As required by project requirements. It is suggested that you make use of React conditional rendering syntax   as used in the Missions and Rockets components


## EXAMPLE
```
        {mission.reserved && (
          <div className="w-64 flex justify-around text-xs">
            <button
              type="button"
              className="text-sm text-red-600 border-red-400 rounded-sm border-2 px-1 py-0.5"
              onClick={() => dispatch(leaveMission(mission.mission_id))}
            >
              {' '}
              leave mission
            </button>
          </div>
        )}
        {!mission.reserved && (
          <div className="w-64 flex justify-around text-xs">
            <button
              type="button"
              className="text-sm text-gray-600 border-gray-400 rounded-sm border-2 px-1 py-0.5"
              onClick={() => dispatch(joinMission(mission.mission_id))}
            >
              {' '}
              Join Mission
            </button>
          </div>
        )}
```

The above is an example and may need to be adapted to your project.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti @AlierPM @Bantealem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navbar/Navbar.js (hunk 1 above)
Comment (Loltolo-Lesapiti) [reply src/components/navbar/Navbar.js]: Thank you for the review. Resolved.

**Review by topeogunleye — APPROVED** [review]
### Hi @Loltolo-Lesapiti @AlierPM @Bantealem, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
image
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlierPM** [comment]
@topeogunleye Thanks for the very constructive review

**Comment by topeogunleye** [comment]
> @topeogunleye Thanks for the very constructive review

You're welcome 😊

**Pull request merged**